tests/python_test.py

```diff
@@ -33,8 +33,14 @@
     # Large offset.
     (LARGE_FILE_SIZE, LARGE_FILE_SIZE - LOCKSPACE_SIZE),
 ])
-def test_write_lockspace(tmpdir, sanlock_daemon, size, offset):
-    path = str(tmpdir.join("lockspace"))
+@pytest.mark.parametrize("filename,encoding", [
+    (u"ascii", None),
+    (u"\u05d0", None),
+    ("ascii", None),
+    (u"\u05d0", "utf-8"),
+])
+def test_write_lockspace(tmpdir, sanlock_daemon, size, offset, filename, encoding):
+    path = util.generate_path(tmpdir, filename, encoding)
     util.create_file(path, size)
```

Comment:
Can we improve util.create_file() to handle also encodings?

Maybe by using a fixfure?

Possible usage:

    def test_foo(temp_file):
        temp_file.create(size=1 * MiB, name=u"\u05d0", encoding=None)

Will create file named u"\u05d0" in tmpdir.

    def test_foo(temp_file):
        temp_file.create(size=1 * MiB, name=u"\u05d0", encoding="utf-8")

Will do the same using bytes.

The tests can use temp_file.path to access the path.

Reply:
IMO this could be confusing, mixing the path creation part with the file creation part. If we consolidate then "encoding" could be mistakenly thought as the file contents encoding.